src/qt/transactionfilterproxy.cpp

```diff
@@ -51,8 +52,11 @@ bool TransactionFilterProxy::filterAcceptsRow(int sourceRow, const QModelIndex &
         return false;
     if(datetime < dateFrom || datetime > dateTo)
         return false;
-    if (!address.contains(addrPrefix, Qt::CaseInsensitive) && !label.contains(addrPrefix, Qt::CaseInsensitive))
+    if (!address.contains(m_search_string, Qt::CaseInsensitive) &&
```

Comment:
BTW, back to `if (!m_search_string.isEmpty()) { ... }`, it can also save querying the model for those properties:
```cpp
if (!m_search_string_lowered_case.isEmpty()) {
    QString address = ...toLowerCase();
    ...
    return ...;

}

Reply:
Sounds like more complication for little value.

Reply:
Probably, maybe @lclc can bench the change in #11015 for his use case.